src/net/NodeHttpClient.ts

```diff
@@ -86,6 +87,8 @@ class NodeHttpClient extends HttpClient {
 }
 
 class NodeHttpClientResponse extends HttpClientResponse {
+  _res: any;
```

Comment:
```suggestion
  _res: Response;
```

Reply:
This doesn't seem to work with how we use `_res`, reverting change for now.